src/sonic-yang-models/yang-models/sonic-ntp.yang

```diff
@@ -0,0 +1,111 @@
+module sonic-ntp {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-system-ntp";
+    prefix ntp;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-mgmt-vrf {
+        prefix mvrf;
+    }
+
+    import sonic-portchannel {
+        prefix po;
+    }
+
+    import sonic-vlan {
+        prefix vlan;
+    }
+
+    import sonic-loopback-interface {
+        prefix lo;
+    }
+
+    import sonic-mgmt-port {
+        prefix mprt;
+    }
+    
+    description
+        "NTP yang Module for SONiC OS";
+
+    revision 2020-04-25 {
+        description
+            "First revision";
+    }
+
+    container sonic-ntp {
+
+        container NTP {
+
+            description "Global NTP part of config_db.json";
+
+            list NTP_LIST {
+                
+                key "global_key";
+
+                leaf global_key {
```

Comment:
check if this can be a container.

Reply:
moved to container instead of list as there is only one global NTP configuration.